subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5_ppi.h

```diff
@@ -125,6 +125,44 @@ static inline void hal_radio_enable_on_tick_ppi_config_and_enable(u8_t trx)
 #define HAL_RADIO_END_TIME_CAPTURE_PPI_TASK \
 	((u32_t)&(EVENT_TIMER->TASKS_CAPTURE[2]))
 
+/* Clear event timer on Radio end:
+ * wire the RADIO EVENTS_END event to the
+ * EVENT_TIMER TASKS_CLEAR task.
+ *
+ * Note: in nRF52X PPI 5 is forked for both capturing and clearing timer
```

Comment:
Effectively, the macros used for the FORKED task are the same as for the PPI for capturing the END time, so we will experience some minor performance degradation, since we will be trying to set the EEP register twice, with the same value.
This is the small price to pay, to get consistency with the rest of PPI configuration, and ease-of-reading.